go/vt/mysqlctl/replication.go

```diff
@@ -69,8 +74,8 @@ func changeMasterArgs(params *sqldb.ConnParams, masterHost string, masterPort in
 }
 
 // parseSlaveStatus parses the common fields of SHOW SLAVE STATUS.
-func parseSlaveStatus(fields map[string]string) *proto.ReplicationStatus {
-	status := &proto.ReplicationStatus{
+func parseSlaveStatus(fields map[string]string) proto.ReplicationStatus {
```

Comment:
Personally, I'd favor making it consistent by doing pass-by-reference everywhere. Although it's a small structure, it may grow in the future, and we've already been bit by something like this once, so it makes me cautious...

Not a blocker though.